.github/workflows/lint.yml

```diff
@@ -100,6 +100,10 @@ jobs:
           # See https://github.com/shadow/shadow/issues/2166
           ref: ${{ github.event.pull_request.head.sha }}
 
+      # Our rust build scripts require libglib.
+      - name: Install system dependencies
+        run: sudo apt-get install -y libglib2.0-dev
```

Comment:
Does clippy run build scripts? I'm guessing so if this is failing, but it seems weird to me.

Reply:
It seems so. I think it's integrated with the compiler so it sort of makes sense?